helpers/batch_submit_PBS.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+#
+
+### Job Name (will be used as prefix later on!)
+#PBS -N icar_batch_run
+### Project code
+#PBS -A P48500028
+#PBS -l walltime=00:15:00
+#PBS -q regular
+### Merge output and error files
+#PBS -o job_output/log.out
+### job error file (PBS will not allow use of ${PBS_JOBID} here? )
+#PBS -e job_output/log.err 
+### Select X nodes with 36 CPUs each for a total of 72 MPI processes
+#PBS -l select=1:ncpus=36:mpiprocs=36:ompthreads=1
+
+### PBS options for automation: https://gif.biotech.iastate.edu/submitting-dependency-jobs-using-pbs-torque
+
+# otherwise xarray is not available:
+module load conda/latest
+source /glade/u/apps/opt/conda/bin/activate
+
+# echo ${PBS_JOBID::7}
+
+# Set OpenMP variables
+export OMP_NUM_THREADS=1
+# export MP_TASK_AFFINITY=core:$OMP_NUM_THREADS
+
+# the easy way
+# icar icar_options.nml
+
+# the complex way (allows a continuous sequence of jobs)   
+PREFIX=$PBS_JOBNAME
+
+# it is useful to keep all other filenames relative to $PREFIX
+# note that this is not required anywhere though
+OUTDIR=$PREFIX
+OPTFILE=${PREFIX}_options.nml  
+BATCHFILE=${PREFIX}_batch_submit.sh
+TEMPLATE=${PREFIX}_template.nml
+
+# specify the location of the icar executable to use:
+EXE=/glade/u/home/bkruyt/usr/local/bin/icar_dbs
```

Comment:
We should try to avoid references to specific user directories if possible.  I'd suggest making this something like 
`EXE=${HOME}/bin/icar` and letting people update it as necessary